.github/workflows/build.yml

```diff
@@ -302,8 +302,8 @@ jobs:
         uses: actions/checkout@v4
       - name: Install rust
         run: |
-          rustup install 1.88
-          rustup default 1.88
+          rustup install stable
```

Comment:
We pinned this version to avoid exactly these kinds of stable-to-stable output changes. I propose `1.89` here.